docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -226,9 +234,8 @@ When you submit a documentation pull request, automated tests are run. Those sam
 
 .. code-block:: bash
 
-  make clean &&
```

Comment:
The sentence below references `make clean` as well, which no longer exists in the root directory (although there is one in the `docs/docsite` directory).

Reply:
nice catch, thx again